sources/fabricFrameSource.js

```diff
@@ -35,7 +36,7 @@ async function mergeFrames({ width, height, framesRaw }) {
     const ctx2 = canvas2.getContext('2d');
     // https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData
     // https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/putImageData
-    ctx2.putImageData(new nodeCanvas.ImageData(Uint8ClampedArray.from(frameRaw), width, height), 0, 0);
+    ctx2.putImageData(new nodeCanvas.ImageData(toArrayInteger(frameRaw), width, height), 0, 0);
```

Comment:
I don't see this function or any merging frames logic in version 0.14.0

Reply:
https://github.com/mifi/editly/blob/65551e79c99933ea5b5684436ee1b0366306a10d/sources/fabric.js#L77